.github/workflows/run-magento-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: Magento2 Build & Run Unit Tests
+
+on: 
+  push:
+    branches-ignore: 
+      - 'master'
+      - 'rc/*/*/*'
+      
+jobs:
+  magento2-build:
+    runs-on: ubuntu-latest
+    name: Magento2 Build & Run Unit Tests
+    
+    services:
+      mysql:
+        image: docker://mysql:8.0
+        env:
+          MYSQL_ROOT_PASSWORD: magento
+          MYSQL_DATABASE: magento
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      opensearch:
+        image: ghcr.io/mad-i-t/magento-opensearch:2.5.0
+        ports:
+          - 9200:9200
+        options: -e="discovery.type=single-node" -e "plugins.security.disabled=true" -e "plugins.security.ssl.http.enabled=false" --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=10
+     
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        path: Cardknox
+
+    - name: Install Magento
+      uses: MAD-I-T/magento-actions@v3.20
+      with:
+        process: 'install-mage-os'
+        magento_version: 2.4.5
+        no_push: 1
+    
+    - name: Install Dependencies
+      run: |
+        cd magento
+        sudo composer install
+
+    - name: Create Directories for Cardknox Module
+      run: |
+        cd magento
+        cd app
+        sudo mkdir code
+        cd code
+        sudo mkdir CardknoxDevelopment
+        cd CardknoxDevelopment         
+      
+    - name: Move Cardknox Module
+      run: |
+        sudo mv Cardknox magento/app/code/CardknoxDevelopment
+        cd magento/app/code/CardknoxDevelopment
```

Comment:
Do we need this cd? We don't seem to be running anything in that directory.

Reply:
No, took both those out and got successful tests here: https://github.com/cardknox/magento2_cardknox/actions/runs/4575957440